src/test/java/org/takes/tk/TkSlf4jTest.java

```diff
@@ -44,9 +44,6 @@
  * @todo #712:30min Prepare implementation for empty response body test and
  *  unignore returnsAnEmptyResponseBody test to fix such error which was
  *  reported in #712.
- * @todo #712:30min Refactor this class to reduce the data abstraction coupling
- *  in order to get rid of the checkstyle suppression of
- *  ClassDataAbstractionCouplingCheck.
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
```

Comment:
@neonailol it's `ClassDataAbstractionCouplingCheck` checkstyle suppression so far, looks like this puzzle was not resolved